events/eventinput/eventInput.go

```diff
@@ -1,115 +1,135 @@
-package events
+package eventinput
 
 import (
 	"fmt"
 	"github.com/FactomProject/factomd/common/interfaces"
 	"github.com/FactomProject/factomd/events/eventmessages/generated/eventmessages"
-	log "github.com/sirupsen/logrus"
+	"github.com/prometheus/common/log"
```

Comment:
using 3 log packages: github.com/prometheus/common/log, github.com/sirupsen/logrus and github.com/FactomProject/factomd/log

Reply:
Reverted in next commit, I only want logrus but IJ is often dropping import lines without asking me.